spec/models/class_member_spec.rb

```diff
@@ -4,7 +4,6 @@
 
 RSpec.describe ClassMember do
   before do
-    stub_user_info_api_for(teacher)
     stub_user_info_api_for(student)
```

Comment:
I noticed there are still a few of these scattered around and wondered why they are still needed?

Reply:
There are still other methods that call the User Info API, notably:

- `ClassMember`
  - `.students`
  - `.with_students`
  - `#with_student`
- `Lesson`
  - `.users`
  - `.with_users`
  - `#with_user`
- `Project`
  - `.users`
  - `.with_users`
  - `#with_user`
- `SchoolClass`
  - `.teachers`
  - `.with_teachers`
  - `#with_teacher`

So anywhere that we're calling these methods will require us to stub the User Info API. Ideally I'd prefer the stubbing to be in the example that requires it (rather than in a `before` block which sets it irrespective of whether the example needs it or not) but I felt that was a change too far for this PR.

Does that make sense?